test/dynamo/test_dynamo.py

```diff
@@ -154,14 +156,22 @@ def test_simple_model(self):
   # then back to the original device.
   @unittest.skipIf(xr.device_type() != "CUDA" or not torch.cuda.is_available(),
                    f"GPU tests should only run on GPU devices.")
-  def test_simple_model_automoves_tensors(self):
-    x = torch.tensor(100.0).to(device="cuda")
-    y = torch.tensor(200.0).to(device="cuda")
+  @parameterized.parameters(
+      "0",
+      "1",
+  )
+  def test_simple_model_automoves_tensors(self, zero_copy_enabled):
+    os.environ.update({
+        xenv.ZERO_COPY_ENABLED: zero_copy_enabled,
+    })
+    x = torch.tensor(100.0).to(device="cuda:0")
+    y = torch.tensor(200.0).to(device="cuda:0")
```

Comment:
shouldn't you check that output is also on `cuda:0`?

Reply:
and somehow verified that computation is run using dynamo not fallback so something

Reply:
> shouldn't you check that output is also on cuda:0?

The test already checks that https://github.com/pytorch/xla/blob/3f54fa21613419680b7c3478ac2320a6612ffa62/test/dynamo/test_dynamo.py#L169

> somehow verified that computation is run using dynamo not fallback so something

The test checks tracing is skipped in following runs https://github.com/pytorch/xla/blob/3f54fa21613419680b7c3478ac2320a6612ffa62/test/dynamo/test_dynamo.py#L172-L175. Do you think it's enough?

Reply:
Since we are looking for fallbacks, what about using `torch_xla._XLAC._get_executed_fallback_ops`?

Reply:
SG. I added a check `self.assertEqual(torch_xla._XLAC._get_executed_fallback_ops(), [])`